WordPress/src/main/java/org/wordpress/android/ui/prefs/MyProfileActivity.java

```diff
@@ -137,21 +139,17 @@ private void updateLabel(WPTextView textView, String text) {
                                                        final String hint,
                                                        final WPTextView textView,
                                                        final boolean isMultiline) {
+        mTextView = textView;
```

Comment:
This assignment seems misplaced and will cause a bug: The TextView that eventually will get updated (by the callback) will always be `mAboutMe` since that's the last one be assigned to mTextView.

Perhaps you were meaning to have this assignment inside the `OnClickListener`. In any case, I would personally prefer to not use a global state variable for this. Instead, I would suggest passing some form of a unique key to `ProfileInputDialogFragment.newInstance` and have it passed back in `onSuccessfulInput` where the caller (MyProfileActivity in this case) would use it to identify which TextView to update. Hope that makes sense.